README.rst

```diff
@@ -104,12 +104,14 @@ packaged in "providers".
 .. code:: python
 
     from faker import Faker
-    from faker.providers import internet
 
     fake = Faker()
-    fake.add_provider(internet)
 
     print(fake.ipv4_private())
+    print(fake.address())
+
+.. note::
+    The `add_provider` method is not necessary unless you are using `external providers`_ - the bundled providers are already available out of the box.
```

Comment:
This render incorrectly at GitHub and referencing `add_provider` without it being shown anywhere does not make much sense IMHO.

Reply:
@stefan6419846 yes you are right, I've updated the rendering part as well as added reference to `add_provider` method